compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ClosureGenerator.java

```diff
@@ -631,6 +631,7 @@ private BLangInvocation getInvocation(BInvokableSymbol symbol) {
         BLangInvocation funcInvocation = (BLangInvocation) TreeBuilder.createInvocationNode();
         funcInvocation.setBType(symbol.retType);
         funcInvocation.symbol = symbol;
+        funcInvocation.pos = symbol.pos;
```

Comment:
Doesn't this set pos of the generated closure to the invocation? That isn't correct, right? Invocation is in the current module.